### ISSUE TITLE
Pandas: SettingWithCopyWarning

### ISSUE THREAD

**Issue by daviddavo** [Grasia/dao-analyzer#63]
Pandas raises a lot of this warning because of some strategies.

For example, in daohaus' `StVotesType`:

https://github.com/Grasia/dao-analyzer/blob/3675e97372881c60806f9595eeed5e1010ba1f2e/src/apps/daohaus/data_access/daos/metric/strategy/st_votes_type.py#L26-L30

What does `dff.loc[:, [self.__DF_DATE, self.__DF_UVOTE]] = dff[[self.__DF_DATE, self.__DF_UVOTE]]` do? Why it is cleaning the df?

**Comment by daviddavo** [comment]
Is this not the same as:

```python
def clean_df(self, df: pd.DataFrame) -> pd.DataFrame:
    return df[[self.__DF_FATE, self.__DF_UVOTE]]
```

**Comment by daviddavo** [comment]
It seems I only removed the ones with tests, but there are a lot more
